src/schema/v2/ArtworkImport/createArtworkImportMutation.ts

```diff
@@ -15,11 +15,19 @@ import { ArtworkImportType } from "./artworkImport"
 
 const SuccessType = new GraphQLObjectType<any, ResolverContext>({
   name: "CreateArtworkImportSuccess",
-  isTypeOf: (data) => data.id,
+  isTypeOf: (data) => !!data.id || !!data.queued,
   fields: () => ({
     artworkImport: {
```

Comment:
When you create an import with the async flag, `artworkImport` will come back as null (since we don't have it yet), but `queued: true` will come back.

So Volt can have the mutation payload adjusted to include `queued` - and the `onComplete` of the mutation can be adjusted to check for a null import/successful queued import - which means it should listen on the websocket for the actual import id. When it receives the import id, it then needs to fetch the import and can continue.